src/pages/Proposal/Description/index.tsx

```diff
@@ -1,15 +1,26 @@
 /** @jsxImportSource @emotion/react */
 import React from 'react';
+import ReactMarkdown from 'react-markdown';
```

Comment:
We should wrap this as a component. I've done that in this PR - https://github.com/VenusProtocol/venus-protocol-interface/pull/641